src/Todo.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './App.css';
+import TodoItem from './TodoItem';
+
+class Todo extends Component {
```

Comment:
better to call it `TodoList` instead of  `Todo` as Todolist give better understanding.